src/amo/reducers/users.js

```diff
@@ -115,6 +115,50 @@ export const fetchUserAccount = ({
   };
 };
 
+type CancelEditUserAccountParams = {|
```

Comment:
I don't understand what `CANCEL_EDIT_USER_ACCOUNT` is for. When would it be called? Can a "user account edit" be cancelled by a user? Can it be cancelled by a saga?

Reply:
It would get cancelled by failure. If you see the saga patch (upcoming) then yeah: because we set an `isEditing` state to `true` during our `editUserAccountApi` call, we need a way to set `isEditing` to `false` if the request failed.

Happy path is:
1. Click "Submit edits"
2. `EDIT_USER_ACCOUNT` dispatched
3. `editUserAccount` API call made
4. API call finishes and saga dispatches `LOAD_USER_ACCOUNT` with new user info
5. `LOAD_USER_ACCOUNT` sets `isEditing` to `false`

Sad path is:
1. Click "Submit edits"
2. `EDIT_USER_ACCOUNT` dispatched
3. `editUserAccount` API call made
4. API call fails/returns error and saga dispatches `SET_ERROR` with error and `CANCEL_EDIT_USER_ACCOUNT`
5. `LOAD_USER_ACCOUNT` is never called, so `CANCEL_EDIT_USER_ACCOUNT` sets `isEditing` to `false`